ui/page/info/wallet_settings_page.go

```diff
@@ -98,6 +119,40 @@ func (pg *WalletSettingsPage) layoutMobile(gtx layout.Context, body layout.Widge
 	return components.UniformMobile(gtx, false, false, body)
 }
 
+func (pg *WalletSettingsPage) clickableRow(gtx C, row row) D {
+	return row.clickable.Layout(gtx, func(gtx C) D {
+		return layout.Inset{Top: values.MarginPadding15, Bottom: values.MarginPaddingMinus5}.Layout(gtx, func(gtx C) D {
+			return pg.subSection(gtx, row.title, func(gtx C) D {
+				return layout.Flex{}.Layout(gtx,
+					layout.Rigid(row.label.Layout),
+					layout.Rigid(func(gtx C) D {
+						ic := row.icon
+						ic.Color = pg.Theme.Color.Gray3
+						return ic.Layout(gtx, values.MarginPadding22)
+					}),
+				)
+			})
+		})
+	})
+}
+
+func (pg *WalletSettingsPage) subSection(gtx C, title string, body layout.Widget) D {
+	return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+		return layout.Flex{}.Layout(gtx,
+			layout.Rigid(pg.subSectionLabel(title)),
+			layout.Flexed(1, func(gtx C) D {
+				return layout.E.Layout(gtx, body)
+			}),
+		)
+	})
+}
+
+func (pg *WalletSettingsPage) subSectionLabel(title string) layout.Widget {
+	return func(gtx C) D {
+		return pg.Theme.Body1(title).Layout(gtx)
+	}
+}
```

Comment:
```suggestion
func (pg *WalletSettingsPage) subSection(gtx C, title string, body layout.Widget) D {
	return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
		return layout.Flex{}.Layout(gtx,
			layout.Rigid(pg.Theme.Body1(title).Layout),
			layout.Flexed(1, func(gtx C) D {
				return layout.E.Layout(gtx, body)
			}),
		)
	})
}
```

this `subSectionLabel` was only used once.